diff --git a/cantata/trunk/PKGBUILD b/cantata/trunk/PKGBUILD
index 7c2b3f73703..bd814971250 100644
--- a/cantata/trunk/PKGBUILD
+++ b/cantata/trunk/PKGBUILD
@@ -12,7 +12,7 @@ pkgdesc='Qt5 client for the music player daemon (MPD)'
 arch=(x86_64)
 url='https://github.com/CDrummond/cantata'
 license=(GPL)
-depends=(libmtp libmusicbrainz5 mpg123 vlc taglib-extras media-player-info libcdio-paranoia udisks2)
+depends=(libmtp libmusicbrainz5 mpg123 taglib-extras media-player-info libcdio-paranoia udisks2)
 optdepends=('perl-uri: dynamic playlist' 'mpd: playback' 'ffmpeg: ReplayGain support' 'libebur128: ReplayGain support')
 makedepends=(cmake qt5-tools ffmpeg libebur128)
 source=("https://github.com/CDrummond/cantata/releases/download/v$pkgver/$pkgname-$pkgver.tar.bz2")
@@ -26,7 +26,8 @@ build() {
   cd build
   cmake ../$pkgname-$pkgver \
     -DCMAKE_INSTALL_PREFIX=/usr \
-    -DENABLE_TOUCH_SUPPORT=ON
+    -DENABLE_TOUCH_SUPPORT=ON \
+    -DENABLE_LIBVLC=OFF
   make
 }
 
