diff --git a/linux-zen/trunk/PKGBUILD b/linux-zen/trunk/PKGBUILD
index 0e25a5fcdb7..d2eff1d0baf 100644
--- a/linux-zen/trunk/PKGBUILD
+++ b/linux-zen/trunk/PKGBUILD
@@ -54,6 +54,7 @@ prepare() {
   # add latest fixes from stable queue, if needed
   # http://git.kernel.org/?p=linux/kernel/git/stable/stable-queue.git
 
+  zcat /proc/config.gz > ../config
   cat ../config - >.config <<END
 CONFIG_LOCALVERSION="${_kernelname}"
 CONFIG_LOCALVERSION_AUTO=n
@@ -71,7 +72,7 @@ END
   # load configuration
   # Configure the kernel. Replace the line below with one of your choice.
   #make menuconfig # CLI menu for configuration
-  #make nconfig # new CLI menu for configuration
+  make nconfig # new CLI menu for configuration
   #make xconfig # X-based configuration
   #make oldconfig # using old config from previous kernel version
   # ... or manually edit .config
