diff --git a/linux-zen/trunk/PKGBUILD b/linux-zen/trunk/PKGBUILD
index 8ba1669cf7a..22ca554573d 100644
--- a/linux-zen/trunk/PKGBUILD
+++ b/linux-zen/trunk/PKGBUILD
@@ -25,6 +25,7 @@ source=(
   '60-linux.hook'  # pacman hook for depmod
   '90-linux.hook'  # pacman hook for initramfs regeneration
   'linux.preset'   # standard config files for mkinitcpio ramdisk
+  'unfuck_MuQSS_for_4.14.15+.patch::https://github.com/ckolivas/linux/commit/25849740d77dfc089fdbfb53623e50d38a972aff.patch'
 )
 validpgpkeys=(
   'ABAF11C65A2970B130ABE3C479BE3E4300411886'  # Linus Torvalds
@@ -40,7 +41,8 @@ sha256sums=('f81d59477e90a130857ce18dc02f4fbe5725854911db1e7ba770c7cd350f96a7'
             'a85bd6e7ff862148b10751a064c710aca6eb9c39a0ba8c06363c299def87d30f'
             'ae2e95db94ef7176207c690224169594d49445e04249d2499e9d2fbc117a0b21'
             '75f99f5239e03238f88d1a834c50043ec32b1dc568f2cc291b07d04718483919'
-            'ad6344badc91ad0630caacde83f7f9b97276f80d26a20619a87952be65492c65')
+            'ad6344badc91ad0630caacde83f7f9b97276f80d26a20619a87952be65492c65'
+            'e0f40fc665ca9fa3cd1f3c9055aa11ea3d5f2790c7c9face69254a24ef27ec04')
 
 _kernelname=${pkgbase#linux}
 
@@ -58,8 +60,9 @@ prepare() {
 
   # add zen patch
   patch -p1 -i ../${_zenpatch}
+  patch -Np1 -i ../unfuck_MuQSS_for_4.14.15+.patch
 
-  cp -Tf ../config .config
+  zcat /proc/config.gz > .config
 
   if [ "${_kernelname}" != "" ]; then
     sed -i "s|CONFIG_LOCALVERSION=.*|CONFIG_LOCALVERSION=\"${_kernelname}\"|g" ./.config
@@ -78,7 +81,7 @@ prepare() {
   # load configuration
   # Configure the kernel. Replace the line below with one of your choice.
   #make menuconfig # CLI menu for configuration
-  #make nconfig # new CLI menu for configuration
+  make nconfig # new CLI menu for configuration
   #make xconfig # X-based configuration
   #make oldconfig # using old config from previous kernel version
   # ... or manually edit .config
