diff --git a/mpv/trunk/PKGBUILD b/mpv/trunk/PKGBUILD
index 36e5a628915..5f34b159551 100644
--- a/mpv/trunk/PKGBUILD
+++ b/mpv/trunk/PKGBUILD
@@ -15,7 +15,7 @@ license=('GPL3')
 url='https://mpv.io/'
 depends=('desktop-file-utils' 'ffmpeg' 'hicolor-icon-theme' 'jack' 'lcms2'
          'libarchive' 'libcaca' 'libcdio-paranoia' 'libdvdnav' 'libgl' 'libva'
-         'libxinerama' 'libxkbcommon' 'libxrandr' 'libxss' 'libxv' 'lua52'
+         'libxinerama' 'libxkbcommon' 'libxrandr' 'libxss' 'libxv' 'luajit'
          'rubberband' 'smbclient' 'uchardet' 'vulkan-icd-loader' 'wayland'
          'xdg-utils' 'shaderc' 'mujs' 'libplacebo')
 makedepends=('mesa' 'python-docutils' 'ladspa' 'wayland-protocols'
@@ -43,7 +43,8 @@ build() {
     --enable-dvdnav \
     --enable-libarchive \
     --enable-libmpv-shared \
-    --enable-libsmbclient
+    --enable-libsmbclient \
+    --lua=luajit
 
   ./waf build
 }
