diff --git a/mpv/trunk/PKGBUILD b/mpv/trunk/PKGBUILD
index 4fbd60640e2..c48ab2357eb 100644
--- a/mpv/trunk/PKGBUILD
+++ b/mpv/trunk/PKGBUILD
@@ -14,7 +14,7 @@ license=('GPL3')
 url='https://mpv.io/'
 depends=('desktop-file-utils' 'ffmpeg' 'hicolor-icon-theme' 'libjack.so' 'lcms2'
          'libarchive' 'libcaca' 'libcdio-paranoia' 'libdvdnav' 'libgl' 'libva'
-         'libxinerama' 'libxkbcommon' 'libxrandr' 'libxss' 'libxv' 'lua52'
+         'libxinerama' 'libxkbcommon' 'libxrandr' 'libxss' 'libxv' 'luajit'
          'librubberband.so' 'smbclient' 'uchardet' 'vulkan-icd-loader' 'wayland'
          'xdg-utils' 'shaderc' 'mujs' 'libplacebo' 'libplacebo.so' 'sndio')
 makedepends=('mesa' 'python-docutils' 'ladspa' 'wayland-protocols'
@@ -37,7 +37,8 @@ build() {
     --enable-libmpv-shared \
     --enable-libsmbclient \
     --enable-sndio \
-    --disable-build-date
+    --disable-build-date \
+    --lua=luajit
 
   waf build
 }
