diff --git a/pinentry/trunk/PKGBUILD b/pinentry/trunk/PKGBUILD
index 1806aaa3908..4ed2629aafe 100644
--- a/pinentry/trunk/PKGBUILD
+++ b/pinentry/trunk/PKGBUILD
@@ -10,19 +10,26 @@ url='https://gnupg.org/related_software/pinentry/'
 arch=('x86_64')
 license=('GPL')
 depends=('ncurses' 'libcap' 'libassuan' 'libsecret')
-makedepends=('gtk2' 'qt5-base' 'gcr')
-optdepends=('gtk2: gtk2 backend'
-            'qt5-base: qt backend'
-            'gcr: gnome3 backend')
+makedepends=(qt5-base)
+optdepends=('qt5-base: qt backend')
 validpgpkeys=('D8692123C4065DEA5E0F3AB5249B39D24F25E3B6')
 source=("https://www.gnupg.org/ftp/gcrypt/${pkgname}/${pkgname}-${pkgver}.tar.bz2"{,.sig}
         'pinentry')
 sha256sums=('68076686fa724a290ea49cdf0d1c0c1500907d1b759a3bcbfbec0293e8f56570'
             'SKIP'
-            '4f51d67c8cf92f2bb535e07d2ba7dc8972535dae077c678102a34fb87f3b3a9c')
+            'SKIP')
 
 backup=('usr/bin/pinentry')
 
+prepare() {
+       cd "${srcdir}/${pkgname}-${pkgver}"
+       cat << EOS > ../pinentry
+#!/bin/bash
+test -e /usr/lib/libQt5Widgets.so.5 && exec /usr/bin/pinentry-qt "\$@"
+exec /usr/bin/pinentry-curses "\$@"
+EOS
+}
+
 build() {
 	cd "${srcdir}/${pkgname}-${pkgver}"
 	./configure \
@@ -30,11 +37,9 @@ build() {
 		--enable-pinentry-tty \
 		--enable-pinentry-curses \
 		--enable-fallback-curses \
-		--enable-pinentry-emacs \
-		--enable-pinentry-gtk2 \
-		--enable-pinentry-gnome3 \
 		--enable-pinentry-qt \
 		--enable-libsecret \
+		--enable-pinentry-qt5 \
 
 	make
 }
